src/mlpack/methods/ann/layer/layer_types.hpp

```diff
@@ -249,7 +250,8 @@ using MoreTypes = boost::variant<
         PositionalEncoding<arma::mat, arma::mat>*,
         ISRLU<arma::mat, arma::mat>*,
         BicubicInterpolation<arma::mat, arma::mat>*,
-        NearestInterpolation<arma::mat, arma::mat>*
+        NearestInterpolation<arma::mat, arma::mat>*,
+        GroupNorm<arma::mat, arma::mat>*,
```

Comment:
```suggestion
        GroupNorm<arma::mat, arma::mat>*
```

Remove the extra `,` to fix the build issue.